deploy/webhook.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-webhook-service
+  namespace: open-cluster-management
+spec:
+  ports:
+  - port: 443
+    protocol: TCP
+    targetPort: 9443
+  selector:
+    control-plane: controller-manager
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: webhook-serving-cert
+  namespace: open-cluster-management
+spec:
+  dnsNames:
+  - webhook-webhook-service.open-cluster-management.svc
+  - webhook-webhook-service.open-cluster-management.svc.cluster.local
+  issuerRef:
+    kind: Issuer
+    name: webhook-selfsigned-issuer
+  secretName: webhook-server-cert
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: webhook-selfsigned-issuer
+  namespace: open-cluster-management
+spec:
+  selfSigned: {}
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  annotations:
+    cert-manager.io/inject-ca-from: open-cluster-management/webhook-serving-cert
+  name: webhook-validating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: webhook-webhook-service
+      namespace: open-cluster-management
+      path: /validate-policy-open-cluster-management-io-v1-policy
+  failurePolicy: Fail
```

Comment:
Like we had talked about, I think it's better to just ignore the failure for now since the Propagator will eventually catch the issue. This webhook is just providing immediate feedback for a better user experience but we don't want it to block policy creation if it's down.

Reply:
Ok I will change it to `Ignore`